dashboard/db/migrate/20170822201210_backfill_authorized_teacher_user_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class BackfillAuthorizedTeacherUserPermissions < ActiveRecord::Migration[5.0]
+  def up
+    Plc::UserCourseEnrollment.find_each do |enrollment|
+      UserPermission.create(permission: UserPermission::AUTHORIZED_TEACHER, user_id: enrollment.user_id)
+    end
+
+    Cohort.find_each do |cohort|
+      cohort.teachers.each do |teacher|
+        UserPermission.create(permission: UserPermission::AUTHORIZED_TEACHER, user_id: teacher.id)
+      end
+    end
+  end
+
+  def down
+    UserPermission.where(permission: UserPermission::AUTHORIZED_TEACHER).destroy_all
```

Comment:
I wasn't sure what would happen with the default `down` method, but deleting manually created permissions with this `down` would be a concern. I'll make that change.